extra/resources/SysInfo.in

```diff
@@ -120,7 +120,7 @@ Can be one of: B, K, M, G, T, P (case-insensitive)
 </content>
 </parameter>
 
-<parameter name="min_disk_free">
+<parameter name="min_disk_free" reloadable="1">
```

Comment:
I believe disks is reloadable too. It's only used in SysInfoStats() to set node attributes, and the attributes aren't cleared by a stop action, so I think it's safe to reload.